src/Sylius/Bundle/UiBundle/Resources/views/Grid/_content.html.twig

```diff
@@ -0,0 +1,12 @@
+{% set definition = grid.definition %}
+{% set data = grid.data %}
+
+{% set path = path(app.request.attributes.get('_route'), app.request.attributes.get('_route_params')) %}
+{% set criteria = app.request.query.get('criteria') %}
+
+{{ sylius_template_event('sylius.admin.grid.filters', _context) }}
+
+<div class="ui hidden divider"></div>
+<div class="sylius-grid-wrapper">
+    {{ sylius_template_event('sylius.admin.grid.body', _context) }}
```

Comment:
I would suggest resignation from `admin` namespace in events. These templates are in UIBundle, therefore are used both in shop and admin context
```suggestion
    {{ sylius_template_event('sylius.grid.body', _context) }}
```